docs/docs/reference/crd-reference/taskdefinition.md

```diff
@@ -260,10 +227,11 @@ spec:
                     Any other scripts listed here are silently ignored.
 
                     - **deno example:**
-                      [Example 2: httpRef script for a Deno script](#example-2-httpref-script-for-a-deno-script)
+                      <a href="#example-2-httpref-script-for-a-deno-script">Example 2: httpRef script for a Deno script</a>
                     - **python example:**
-                      [Example 2: httpRef for a python-runtime runner](#example-2-httpref-for-a-python-runtime-runner)
-
+                      <!-- markdownlint-disable MD033 -->
+                      <!-- <a href="#"></a> --><br>
+                      <!-- markdownlint-enable MD033 -->
```

Comment:
Do we need the disabling of MD033 rule? We allowed <a> in the allowed elements [here](https://github.com/keptn/lifecycle-toolkit/pull/3005/files#diff-2a7fead57f7c323860561f88108bea9da390f1fcccc08278680b62b0ff6e1d5bR12)

Reply:
It's for br element

That was MD013 for `a` element

Reply:
but why do we need a <br> element here? Does it make a big difference on the site? I would rather avoid it